sharding-core/sharding-core-preprocessor/src/test/java/org/apache/shardingsphere/core/preprocessor/segment/select/projection/DerivedColumnTest.java

```diff
@@ -49,15 +49,15 @@ public void assertIsNotDerivedColumn() {
     
     @Test
     public void assertIsDerivedColumnName() {
-        assertTrue(DerivedColumn.isDerivedColumnName("AVG_DERIVED_COUNT_"));
-        assertTrue(DerivedColumn.isDerivedColumnName("AVG_DERIVED_SUM_"));
-        assertTrue(DerivedColumn.isDerivedColumnName("ORDER_BY_DERIVED_"));
-        assertTrue(DerivedColumn.isDerivedColumnName("GROUP_BY_DERIVED_"));
-        assertTrue(DerivedColumn.isDerivedColumnName("AGGREGATION_DISTINCT_DERIVED_"));
+        assertTrue(DerivedColumn.isDerivedColumn("AVG_DERIVED_COUNT_"));
+        assertTrue(DerivedColumn.isDerivedColumn("AVG_DERIVED_SUM_"));
+        assertTrue(DerivedColumn.isDerivedColumn("ORDER_BY_DERIVED_"));
+        assertTrue(DerivedColumn.isDerivedColumn("GROUP_BY_DERIVED_"));
+        assertFalse(DerivedColumn.isDerivedColumn("AGGREGATION_DISTINCT_DERIVED_"));
     }
-    
+
     @Test
     public void assertIsNotDerivedColumnName() {
-        assertFalse(DerivedColumn.isDerivedColumnName("OTHER_DERIVED_COLUMN_0"));
+        assertFalse(DerivedColumn.isDerivedColumn("OTHER_DERIVED_COLUMN_0"));
```

Comment:
Please refactor this test cases to make sure that one method only have one test cases for same scenario.

Reply:
I've merged these 2 test cases which refer to the method isDerivedColumn